src/SignalRServiceExtension/TriggerBindings/ServerlessHub.cs

```diff
@@ -41,15 +41,20 @@ public abstract class ServerlessHub : IDisposable
         protected ServerlessHub(IServiceHubContext hubContext = null, IServiceManager serviceManager = null)
         {
             HubName = GetType().Name;
-            hubContext = hubContext ?? StaticServiceHubContextStore.Get().GetAsync(HubName).GetAwaiter().GetResult();
-            _serviceManager = serviceManager ?? StaticServiceHubContextStore.Get().ServiceManager;
+            hubContext = hubContext ?? StaticServiceHubContextStore.Get(Connection).GetAsync(HubName).GetAwaiter().GetResult();
+            _serviceManager = serviceManager ?? StaticServiceHubContextStore.Get(Connection).ServiceManager;
             Clients = hubContext.Clients;
             Groups = hubContext.Groups;
             UserGroups = hubContext.UserGroups;
             _hubContext = hubContext as ServiceHubContext;
             ClientManager = _hubContext?.ClientManager;
         }
 
+        /// <summary>
+        /// The connection string setting name.
+        /// </summary>
+        public string Connection { get; set; } = Constants.AzureSignalRConnectionStringName;
```

Comment:
Feels like this should be `get` only, but then it doesn't implement `IConnectionProvider`, but not sure how important that is

Reply:
I think it is OK.